crates/next-api/src/nft_json.rs

```diff
@@ -112,13 +113,13 @@ async fn apply_includes(
     while let Some(glob_result) = stack.pop_back() {
         // Process direct results (files and directories at this level)
         for entry in glob_result.results.values() {
-            let DirectoryEntry::File(file_path) = entry else {
+            let (DirectoryEntry::File(file_path) | DirectoryEntry::Symlink(file_path)) = entry
+            else {
                 continue;
             };
 
-            let file_path_ref = file_path;
             // Convert to relative path from ident_folder to the file
-            if let Some(relative_path) = ident_folder.get_relative_path_to(file_path_ref) {
+            if let Some(relative_path) = ident_folder.get_relative_path_to(file_path) {
                 result.insert(relative_path);
             }
```

Comment:
Should it also add all realpaths to the nft.json file?

realpath_with_symlinks -> add the result and all symlinks?